.circleci/config.yml

```diff
@@ -60,6 +60,8 @@ jobs:
     # vulnerabilities using pyup.io, and run the check.
     - run: pip3 install --user -U safety
     - run: safety check --bare -r requirements.txt
+    - run: python3 manage.py makemigrations
```

Comment:
Migrations should already be committed to code base and results tested before any deployments, so `makemigrations` should not appear in any deployment processes.

Reply:
Yes noticed my local migration files weren't being committed so reverting this.